src/main/java/org/tikv/common/PDClient.java

```diff
@@ -212,6 +228,115 @@ LeaderWrapper getLeaderWrapper() {
     return leaderWrapper;
   }
 
+  private List<URI> getMemberUrI() {
```

Comment:
This is an odd spelling even for a private API 😅 

```suggestion
  private List<URI> getMemberURI() {
```

Reply:
thanks for pointing out